### PR TITLE
The second day of HW

### PR DESCRIPTION
**Pull request by wh0-1-am**
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
-0
Was there anything that you struggled with?;
-no
Is there anything that you'd like some further information on?;
-no
Roughly how long did it take?
-Around 2 hour or less

### CODE REVIEW

**Comment by wh0-1-am** [comment]
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
0
Was there anything that you struggled with?;
no
Is there anything that you'd like some further information on?;
Yes,Can you give example for each task?. That will be more clear for student, thanks
Roughly how long did it take?
less than 1 hour

**Comment by wh0-1-am** [comment]
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
1
Was there anything that you struggled with?;
no
Is there anything that you'd like some further information on?;
no
Roughly how long did it take?
1 h

File: Sami_Aldahlawi/week_01/day_04/main.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        "Ask again",
+        "No",
+        "I doubt it",
+        "No Way"
+      ];
+      var rand = Math.floor(Math.random() * fortunes.length);
+      return "The " + jobTitle + ", " + name + " of " + locations + ", asks " + question + " ... " + fortunes[rand];
+}
+
+console.log(tellFortune('Moath', 'IA', 'Demman', 'will I win the lottery this week?'));
```
Comment (mfalthaw) [inline Sami_Aldahlawi/week_01/day_04/main.js]: I hope I do!

Comment (wh0-1-am) [reply Sami_Aldahlawi/week_01/day_04/main.js]: I hope so, but don't forget me  (;
Updated the code. You can check it now.
Thanks

**Pull request merged**